sdk/containerregistry/container-registry/swagger/containerregistry.json

```
@@ -1499,22 +1482,21 @@
     "AcrManifests": {
```

Could we add this to get this extensible enum generated for us?

```suggestion
    "TagOrderBy": {
      "type": "string",
      "enum": [
        "time_desc",
        "time_asc"
      ],
      "x-ms-enum": {
        "name": "TagOrderBy",
        "modelAsString": true,
        "values": [
          {
            "value": "time_desc",
            "name": "LastUpdatedOnDescending",
            "description": "Order tags by LastUpdatedOn field, from most recently updated to least recently updated."
          },
          {
            "value": "time_asc",
            "name": "LastUpdatedOnAscending",
            "description": "Order tags by LastUpdatedOn field, from least recently updated to most recently updated."
          }
        ]
      },
    "AcrManifests": {
```